components/sidebar/channel_filter/channel_filter.tsx

```diff
@@ -21,10 +26,28 @@ type State = {
 
 };
 
-export default class ChannelFilter extends React.PureComponent<Props, State> {
-    toggleUnreadFilter = (e?: React.MouseEvent) => {
+class ChannelFilter extends React.PureComponent<Props, State> {
+    componentDidMount() {
+        document.addEventListener('keydown', this.handleUnreadFilterKeyPress);
+    }
+
+    componentWillUnmount() {
+        document.removeEventListener('keydown', this.handleUnreadFilterKeyPress);
+    }
+
+    handleUnreadFilterClick = (e?: React.MouseEvent) => {
         e?.preventDefault();
```

Comment:
I think we should stop propagation 
```suggestion
        e?.preventDefault();
        e?.stopPropagation();
```